generator/typescript/index.d.tstemplate

```diff
@@ -41,8 +41,9 @@ export class DropboxAuth {
    * @param redirectUri A URL to redirect the user to after authenticating.
    *   This must be added to your app through the admin interface.
    * @param code An OAuth2 code.
+   * @returns {Object} An object containing the token and related info (if applicable)
    */
-  getAccessTokenFromCode(redirectUri: string, code: string): Promise<DropboxResponse<string>>;
+  getAccessTokenFromCode(redirectUri: string, code: string): Promise<DropboxResponse<object>>;
```

Comment:
Although this is fine as a quick fix, it would be better if this was a typed object.